### DISCUSSION TITLE
GHB inflow only

### DISCUSSION THREAD

**Discussion by wkitlasten** [MODFLOW-ORG/modflow6#1183] [Ideas]
Estimating mountain block/mountain front recharge in valley aquifers surrounded by steep terrain is difficult, but in most cases we can be sure flow in these areas is into (not out of) the valley aquifer. The GHB is convenient, but can lead to local circulation cells especially in stochastic ensembles.

A relatively simple solution might be to have a "inflow_only" OPTION in the GHB package to prevent water from leaving via the GHB but allowing GHB to provide inflow as water levels drop. Perhaps there is another, better option that already exists?

Apparently GHB was modified in a similar way in this study: https://link.springer.com/article/10.1007/s11269-015-1022-y

"the GHB package was modified in such a way that flow is permitted to occur only in to the model cells based on an inward gradient and not out of it."

**Comment by christianlangevin** [comment]
Thanks for pointing this out, @wkitlasten.  I wasn't aware of this work, but can see potential value in the capability to simulate "topping off" of an aquifer.  This could be done fairly easily now with the [MODFLOW-API](https://github.com/MODFLOW-USGS/modflowapi).  We will consider an "inflow_only" option for GHB.
